### PR TITLE
profiles: force host targets for LLVM also on musl and uclibc

### PR DESCRIPTION
**Pull request by doughdemon**
@mgorny right now I've just copied the message from your commit. Feel free to commit something else along the line instead of this PR.

### CODE REVIEW

**Comment by blueness** [comment]
looks good to me

File: profiles/default/linux/uclibc/arm/package.use.mask [hunk 1]
```diff
@@ -2,6 +2,11 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
+# Michał Górny <mgorny@gentoo.org> (24 Sep 2016)
+# Force the host target to avoid dependency hell
```
Comment (mgorny) [inline profiles/default/linux/uclibc/arm/package.use.mask]: That's package.use.mask ;-P.

**Comment by doughdemon** [comment]
> +# Michał Górny <mgorny@gentoo.org> (24 Sep 2016)
> +# Force the host target to avoid dependency hell
> 
> That's package.use.mask ;-P.

Thanks for spotting this. Should be fixed now.

**Comment by blueness** [comment]
pushed

**Pull request closed without merging**